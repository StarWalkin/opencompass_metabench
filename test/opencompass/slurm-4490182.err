  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]slurmstepd: error: *** JOB 4490182 ON gpua092 CANCELLED AT 2024-08-16T07:22:07 DUE TO TIME LIMIT ***
