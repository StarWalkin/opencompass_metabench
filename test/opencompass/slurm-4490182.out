/u/zzhang27/.conda/envs/opencompass/bin/python3
08/16 06:52:27 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/16 06:52:27 - OpenCompass - INFO - Loading hf_qwen2_7b_instruct: configs/models/qwen/hf_qwen2_7b_instruct.py
08/16 06:52:28 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/16 06:52:28 - OpenCompass - INFO - Current exp folder: outputs/default/20240816_065228
08/16 06:52:28 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/16 06:52:28 - OpenCompass - INFO - Partitioned into 1 tasks.
